frontend/src/App.tsx

```diff
@@ -30,9 +30,9 @@ const App = () => {
   // http://keycloak.6j0.org/auth/realms/STAR%20Voting/.well-known/openid-configuration
   let keycloakBaseUrl;
   if(prodEndpoints.includes(window.location.origin)){
-    keycloakBaseUrl = 'https://keycloak.6j0.org/auth/realms/STAR%20Voting/protocol/openid-connect';
+    keycloakBaseUrl = 'https://52.205.245.149:8443/realms/STAR%20Voting/protocol/openid-connect';
   }else{
-    keycloakBaseUrl = 'https://keycloak.6j0.org/auth/realms/STAR%20Voting%20Dev/protocol/openid-connect';
+    keycloakBaseUrl = 'https://52.205.245.149:8443/realms/STAR%20Voting%20Dev/protocol/openid-connect';
```

Comment:
is this step included in the guide?

Reply:
Yep, I mention "frontend/src/App.tsx" toward the end of the keycloak_configuration doc